tests/integration/create_token.py

```diff
@@ -1,7 +1,11 @@
+from uuid import uuid4
+
+import time
+
 from app.views.dev_mode import generate_token, create_payload
 
 PAYLOAD = {
-    'user': "mci-integration-test",
+    'user_id': "mci-integration-test",
```

Comment:
be consistent with quotes, should be single quotes everywhere really (wonder if there is a lint rule for this?)